### ISSUE TITLE
Picture in Picture support

### ISSUE THREAD

**Issue by Komal2905** [AgoraIO/Basic-Video-Broadcasting#70]
Hi, I need to implement PIP mode on video broadcasting. is this library support this

**Comment by plutoless** [comment]
yes. you only need to layout your views

**Comment by Komal2905** [comment]
@plutoless  How to achieve that in native iOS 
Do you have any idea?
Do we have to use AVPictureInPictureController?

**Comment by ParabellumApp** [comment]
Yes, It would be great! An example of Picture in picture implementation on Agora basic broadcasting demo.
